srv/modules/runners/cephprocesses.py

```diff
@@ -60,11 +69,18 @@ def _status(search, roles):
 
     status = {}
     local = salt.client.LocalClient()
+
+    ignore_roles = [ 'admin', 'master' ]
+
+    for ig_role in ignore_roles:
+      if roles.has_key(ig_role) and roles:
```

Comment:
```py
for ig_role in ignore_roles:
    if not roles:
        # do something because roles dict does not exist
    roles.pop(ig_role, None) # returns None if role doesn't exist & therefore doesn't throw error
```